app/controllers/data_exports_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# Controller actions for Data Exports
+class DataExportsController < ApplicationController
+  before_action :data_export, only: %i[download destroy]
+
+  def index
+    @data_exports = DataExport.where(user: current_user)
+  end
+
+  def new
+    render turbo_stream: turbo_stream.update('export_modal',
+                                             partial: 'new_export_modal',
```

Comment:
**question:** Is the UI (and this partial) coming in a subsequent PR?

Reply:
Yes all UI elements will be tied in with future PRs

Reply:
Ok that's what I figured.  My only comment here is that you should not be referring to UI elements that do not yet exist. Don't worry about it now as I suspect you have another branch with this already.